src/main/java/com/spotify/docker/client/messages/ContainerConfig.java

```diff
@@ -392,6 +400,9 @@ public static Healthcheck create(
       if (retries != null) {
         builder.retries(retries);
       }
+      if (startPeriod != null) {
```

Comment:
@johnflavin It seems to be the pattern regardless if the field is nullable (see retries for example).  I can update if necessary.